Controller/Adminhtml/Import/Ajax.php

```diff
@@ -47,46 +39,55 @@ public function __construct(
-
-        $client = new Client($token, $platform, $pluginVersion);
-    
-        echo json_encode(array(
-            'customer'      => $client->customer($this->customerData->getCustomers($storeId)[0]),
-            'customerBatch' => $client->customerBatch($this->customerData->getCustomers($storeId)),
-            'category'      => $client->category($this->categoryData->getCategories($storeId)[0]),
-            'categoryBatch' => $client->categoryBatch($this->categoryData->getCategories($storeId)),
-            'product'       => $client->product($this->productData->getProducts($storeId)[0]),
-            'productBatch'  => $client->productBatch($this->productData->getProducts($storeId)),
-            'order'         => $client->order($this->orderData->getOrders($storeId)[0]),
-            'orderBatch'    => $client->orderBatch($this->orderData->getOrders($storeId))
-        ));
-        exit;
-
         try {
-            $jsonFactory = $this->resultJsonFactory->create();
-            $result = ['success' => false];
+            $jsonFactory   = $this->resultJsonFactory->create();
+            $result        = ['success' => false];
+            $storeId       = (int)$this->request->getParam('storeId');
+            $chunkId       = (int)$this->request->getParam('chunkId');
+            $totalChunks   = (int)$this->request->getParam('totalChunks');
+            $importStatus  = (string)$this->request->getParam('importStatus');
+            
+            $token         = $this->helper->getApiToken($storeId);
+            $platform      = 'Magento ' . $this->helper->metaData->getEdition() . ' ' . $this->helper->metaData->getVersion();
+            $pluginVersion = $this->helper->moduleList->getOne($this->helper::MODULE_NAME)['setup_version'];
             
-            $chunkId = (int)$this->request->getParam('chunkId');
-            $totalChunks = (int)$this->request->getParam('totalChunks');
+            $client        = new Client($token, $platform, $pluginVersion);
 
-            if ($chunkId == 0) {
-                $this->helper->createActivity($storeId, 'import_start');
-            }
+//            if ($chunkId == 0) {
+//                $this->helper->createActivity($storeId, 'import_start');
+//            }
 
-            // Get orders from the Database
-            $orders = $this->import->getOrders($storeId, $chunkId);
-            // Send orders via API helper method
-            $this->helper->callBatchApi($storeId, $orders, false);
-            $result['success'] = true;
+            // file_put_contents(__DIR__ . 'Request.log', $client->customerBatch($this->import->customerData->getCustomers($storeId, $chunkId, $this->import->chunkItems)), FILE_APPEND); // Used for debug purposes;
 
-            if ($chunkId == $totalChunks - 1) {
-                $this->helper->createActivity($storeId, 'import_end');
+            switch ($importStatus) {
+                case 1:
```

Comment:
why don't you use strings to make this more understandable?

Reply:
Statuses changed customer, category, product, order.